### ISSUE TITLE
Implement Studio UI

### ISSUE THREAD

**Issue by lmiller1990** [cypress-io/cypress#23337]
Part of the Cypress Studio Epic: https://github.com/cypress-io/cypress/issues/23230

AC:

- Ensure Studio UI renders correct in the new Cypress 10 app UI. 
- Use existing UI elements where possible (either from Cypress 9's original experimental Studio feature, or Cypress 10's new design system, whatever is easiest and looks best)
- Make it look decent/usable (need not be perfect on the first pass, since we want to collaborate with design on a unified look/feel. That's part of #23410).

Out of scope:

- Redesign the UI using our new design system

## Screenshots

You can see Studio in action by creating a new project with Cypress 9:

```
mkdir foo
cd foo
yarn init -y
yarn add cypress@9.7.0
echo '{"experimentalStudio": true}' > cypress.json
yarn cypress open
```

Make sure you do the above and you must do `experimentalStudio: true` in the `cypress.json`.

Next, create an empty spec, eg `it('works', () => {})`.

Click the little wand (hover the test description) to activate studio:

<img width="724" alt="Screen Shot 2022-08-17 at 4 10 42 pm" src="https://user-images.githubusercontent.com/19196536/185047306-1f33009e-2bc8-400b-a0ab-6d2876df0eb6.png">

More info here: https://docs.cypress.io/guides/references/cypress-studio#Using-Cypress-Studio

We will _eventually_ reskin it to look like the rest of the App UI. It is still experimental for now, so it's fine to just make it look good enough to ship - we will rework the UI before moving out of experimental.

Here's the specific UI elements we need (it should mostly exist in the code base, to find specifics you might need to check out Cypress 9. You can do it with `git checkout v9.7.0` and look around there).

<img width="1537" alt="Screen Shot 2022-08-17 at 4 00 43 pm" src="https://user-images.githubusercontent.com/19196536/185046460-e52e22a1-30d4-4e45-ba31-df6e99554903.png">

<img width="1537" alt="Screen Shot 2022-08-17 at 4 01 43 pm" src="https://user-images.githubusercontent.com/19196536/185046488-90f4b801-3dba-4b12-a7c9-345a791ec742.png">

<img width="1537" alt="Screen Shot 2022-08-17 at 4 02 21 pm" src="https://user-images.githubusercontent.com/19196536/185046501-19023e90-94a9-4d26-be85-614c3b9147df.png">

<img width="1537" alt="Screen Shot 2022-08-17 at 4 03 36 pm" src="https://user-images.githubusercontent.com/19196536/185046517-847d22d7-b5de-4ead-a658-fa1778551193.png">

<img width="963" alt="Screen Shot 2022-08-17 at 4 04 39 pm" src="https://user-images.githubusercontent.com/19196536/185046533-924cfa24-1bb5-4f82-8dcc-acedef7e8da5.png">

<img width="1537" alt="Screen Shot 2022-08-17 at 4 05 02 pm" src="https://user-images.githubusercontent.com/19196536/185046548-aec6a3ce-2aa7-43be-b8fd-5367f3c0196e.png">

Controls in Command Log when Studio is active:

![image](https://user-images.githubusercontent.com/19196536/185282207-a026f300-05f8-43a2-a6bb-91bc6ec6eb6e.png)

Note: the copy button should copy some text to clipboard, and then the tooltip should change from "copy" to "copied" like in this PR: https://github.com/cypress-io/cypress/pull/16912

**Comment by lmiller1990** [comment]
@astone123 just noticed some additional requirements for the UI, or at least, if we want to keep parity with 9.x (I'll let you and the designer figure out the final UI):

Check this PR (and the screenshot): the **copy** button should copy some text to clipboard, and then the tooltip should change from "copy" to "copied" like in this PR: https://github.com/cypress-io/cypress/pull/16912. You'll have access to the text once my wiring PR is done, so you can assume it's readily available.
